.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Pacifica Proxymod version
```

Comment:
This should be fixed, wrong name `Proxymod`

Reply:
Done!